docs/getting-started/data-views-in-sec.asciidoc

```diff
@@ -0,0 +1,37 @@
+[[data-views-in-sec]]
+= {data-sources-cap} in Elastic Security
+
+The active {data-source} determines what data appears throughout {es-sec} on pages with event or alert data.
+{Data-sources} are defined by the index patterns they include.
+Only data from the {es} {ref}/documents-indices.html[indices], {ref}/data-streams.html[data streams], or {ref}/alias.html[index aliases] specified in the active {data-source} will appear.
+
+IMPORTANT: Custom indices may not be included in the <<default-data-view-security, default {data source}>>. Modify it or create a custom {data-source} to include custom indices.
+
+[discrete]
+== Switch to another {data-source}
+
+You can tell which {data-source} is active by clicking the {data-source-cap} button that appears at the upper right of {es-sec} pages that display data, such as Overview, Alerts, or Hosts.
+To switch to another {data-source}, click `Choose {data-source}` and select one of the options.
+
+image::images/dataview-select-gif.gif[gif of opening the data view selection menu]
+
```

Comment:
We should add a section about filtering the index patterns in a Data View. We should note that only index patterns that have matching indices with data are available for selection. The disabled (grayed out) index patterns do not currently match any indices.
![filtering](https://user-images.githubusercontent.com/6935300/171730149-9f949a74-110e-4b27-9627-b71190c9a568.gif)